bank.rb

```diff
@@ -0,0 +1,45 @@
+$balance = 0
+continue = true
+
+def deposit(amount)
```

Comment:
Let's add all of these methods to a ```Bank``` class, so that they apply to all instances of ```Bank``:
```
class Bank
	attr_reader :balance

	def initialize
		@$balance = 0
	end

	# def deposit(amount), def withdraw(amount) and so on...
end
```